tutorials/tour/_posts/2017-02-13-operators.md

```diff
@@ -10,8 +10,40 @@ num: 30
 next-page: automatic-closures
 previous-page: local-type-inference
 ---
+In Scala, operators are methods. For example, `+` can be called with dot-notation:
+```
+10.+(1)
+```
+
+However, it more closely resembles math if you write it as an infix operator:
+```
+10 + 1
+```
+
+You can also give `+` specific meaning in your class:
+```tut
+class Vector(val x: Double, val y: Double) {
+  def +(that: Vector) = new Vector(this.x + that.x, this.y + that.y)
+}
+
+val vector1 = new Vector(1.0, 1.0)
+val vector2 = new Vector(2.0, 2.0)
+
+val vector3 = vector1 + vector2
+vector3.x  // 3.0
+vector3.y  // 3.0
+```
+The class vector has a method `+` which we used to add vector1 and vector2. When the method has two or more parameters, use a tuple:
+```tut
+class Rectangle(val x: Double, val y: Double) {
+  def changeSize(dx: Int, dy: Int): Rectangle = new Rectangle(x + dx, y + dy)
+}
+
+val rect1 = new Rectangle(1, 2)
+val rect2 = rect1 changeSize (2, 2)  // equivalent to rect1.changeSize(2, 2)
+```
 
-Any method which takes a single parameter can be used as an *infix operator* in Scala. Here is the definition of class `MyBool` which includes methods `and` and `or`:
+Using parentheses, you can build up complex expressions with readable syntax. Here is the definition of class `MyBool` which includes methods `and` and `or`:
```

Comment:
You could maybe stay in the same `Vec` context and use `add` instead of `+` and say that all methods (not just those with symbolic names) can be used with the infix notation.